packages/ckeditor5-image/src/imageinsert/utils.ts

```diff
@@ -17,11 +19,11 @@ import { LabeledFieldView, createLabeledInputText } from 'ckeditor5/src/ui';
  *
  * @returns {Object.<String, module:ui/view~View>} Integrations object.
  */
-export function prepareIntegrations( editor ) {
+export function prepareIntegrations( editor: Editor ): Record<string, View> {
```

Comment:
Maybe this should return sth more specific than `View` ? some dedicated interface?

Reply:
I'm not sure if that can be done. In tests, some integrations are defined as `View`